utils/httpUtils.ts

```diff
@@ -40,15 +60,35 @@ export const get = async (url: string, args: any, context: any) => {
 export const getFullResponse = async (url: string, args: any, context: any) => {
   try {
     const baseURL = process.env.API_URL;
-    const urlPrefix = args.snapshotLinkId ? `/pub/${args.snapshotLinkId}` : "";
-    const response = await fetch(baseURL + urlPrefix + url, {
-      method: "GET",
-      headers: {
-        Cookie: context.request.headers.get("cookie"),
-        "Content-Type": "application/json",
-      },
-    });
-    return response;
+    const nextGenEnabled = await isNextGenEnabled(context);
+    if (nextGenEnabled) {
+      // next gen details
+      const czidServicesToken = await getEnrichedToken(context);
```

Comment:
I wanted to clarify this variable name.  The czid services token is different from the enriched token.  That token is a simple user id token, which we send to rails to "exchange" for the enriched token.  I think we should use the proper names here to avoid confusion on what token is being sent to nextgen.

That said, if we extract this code to the shared util function, we don't really need to worry about it here, and the stub method already uses the accurate name: https://github.com/chanzuckerberg/czid-graphql-federation-server/blob/010967bd3011dd1544a6bad984f4e64e2f3f0b08/utils/callGraphqlSource.ts#L4